browser/src/view/components/Button/Button.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Button.scss';
+
+export const Button = ({ layout, label, Icon, type }) => {
+  return (
+    <button
+      className={`${
+        type === ButtonAttributes.Type.destructive
+          ? 'btn-destructive'
+          : type === ButtonAttributes.Type.primary
+          ? 'btn-primary'
+          : 'btn-secondary'
+      }
+      ${
+        type === ButtonAttributes.Layout.iconAndLabel
+          ? 'btn-icon-label'
+          : type === ButtonAttributes.Layout.iconOnly
+          ? 'btn-icon-only'
+          : 'btn-label-only'
+      }`}
+    >
+      {layout === ButtonAttributes.Layout.iconOnly ||
+      layout === ButtonAttributes.Layout.iconAndLabel
+        ? Icon
+        : null}
```

Comment:
You don't need to use a ternary for conditional rendering.  `&&` will suffice as React won't render most falsy values (except maybe for 0, depending on React version).